velox/functions/sparksql/tests/SubscriptTest.cpp

```diff
@@ -41,42 +41,4 @@ class SubscriptTest : public SparkFunctionBaseTest {
 };
 
 } // namespace
-
-// Spark's subscript ("a[1]") behavior:
```

Comment:
Why are you deleting this test?

Reply:
Because this UT contains outdate function "getarrayitem" and "getmapvalue". And I just removed those two functions in https://github.com/facebookincubator/velox/pull/4095

Reply:
Hi, @mbasmanova would you take a look at https://github.com/facebookincubator/velox/pull/4095.

Reply:
@zhejiangxiaomai I just looked at #4095 and left some comments. Thank you for extracting these changes into a separate PR.